stan/math/rev/core/var.hpp

```diff
@@ -902,6 +902,20 @@ class var_value<
     return var_sub(new vari_sub(vi_->middleCols(start_col, n)));
   }
 
+  /**
+   * Return an Array.
+   */
+  inline auto array() const {
```

Comment:
Drop a basic test or two in: https://github.com/stan-dev/math/blob/develop/test/unit/math/rev/core/var_test.cpp

Doesn't need to be exhaustive.